### ISSUE TITLE
A group / project entity

### ISSUE THREAD

**Issue by RoyWiggins** [mercure-imaging/mercure#23]
Right now there's no concept of a group/project. It would be useful to be able to define, eg, a study and attach rules, processing modules, and destinations to it. This would be useful for grouping related rules together, and also for generating reports on eg the progress of a project (how many incoming cases have been routed by a rule attached to a project, and so on).

**Comment by tblock79** [comment]
Agreed! The "Tags" field on the Rule -> Information tab is a first step in that direction, but the information is not evaluated for anything as of now
